common/persistence/dataInterfaces.go

```diff
@@ -655,6 +686,7 @@ type (
 		GetWorkflowExecution(request *GetWorkflowExecutionRequest) (*GetWorkflowExecutionResponse, error)
 		UpdateWorkflowExecution(request *UpdateWorkflowExecutionRequest) error
 		DeleteWorkflowExecution(request *DeleteWorkflowExecutionRequest) error
+		DeleteSignalRequestedID(request *DeleteWorkflowExecutionSignalRequestedRequest) error
```

Comment:
Why this is a new API but DeleteSignalInfo is part of update?

Reply:
removed